Gems/PhysX/Code/Source/EditorRigidBodyComponent.cpp

```diff
@@ -286,6 +287,12 @@ namespace PhysX
         }
         CreateEditorWorldRigidBody();
 
+        PhysX::EditorColliderComponent* entityColliderComponent = m_entity->FindComponent<PhysX::EditorColliderComponent>();
+        if (entityColliderComponent != nullptr)
+        {
+            entityColliderComponent->ValidateRigidBodyMeshGeometryType();
+        }
+
```

Comment:
You are creating a dependency from EditorRigidBodyComponent with EditorColliderComponent.
Can you use an EBus instead?